### PR TITLE
Adds getListInstance function

### PR DESCRIPTION
**Pull request by cperryk**
Adds `getListInstance`, which returns the name of the list URL or URI that you pass into it.

Distantly related to [Trello](https://trello.com/c/Z0tqynOw/36-add-site-to-site-import-in-clay-cli)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "clay-utils",
-  "version": "1.1.1",
+  "version": "1.2.0",
```
Comment (cperryk) [inline package.json:3]: (This reflects what's on NPM; made a mistake with the release command when I last released)

File: lib/getListInstance/index.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const name = __filename.split('/').pop().split('.').shift(),
+  fn = require('./' + name),
+  expect = require('chai').expect;
+
+describe('getListInstance', () => {
+
+  it('gets instance from uri', function () {
```
Comment (amycheng) [inline lib/getListInstance/index.test.js]: might as well commit fully to using fat arrow syntax

**Review by amycheng — DISMISSED** [review]
is there a reason for adding `shrinkwrap`?

**Comment by cperryk** [comment]
@amycheng Removed fat arrow b/c you shouldn't use them in mocha as mocha passes info to `this`. Shrinkwrap was already here and the changes here are only incrementing the version number following the release.

**Review by amycheng — APPROVED** [review]

**Pull request merged**